CSharp_Course_Project/Program.cs

```diff
@@ -4,9 +4,709 @@ namespace CSharp_Course_Project
+                bool isNumber2 = Int32.TryParse(stringNumber2, out int number2);
+                if (!isNumber1 || !isNumber2)
+                {
+                    Console.WriteLine("Not a numeric values. Try again...");
+                    continue;
+                }
+
+                int result = 0;
+                if (number1 >= 0 && number2 >= 0)
+                {
+                    for (int i = 0; i < number1; i++)
+                    {
+                        result += number2;
+                    }
+                }
+                else if (number1 < 0)
+                {
+                    for (int i = 0; i < -number1; i++)
+                    {
+                        result += -number2;
+                    }
+                }
+                else if (number2 < 0)
+                {
+                    for (int i = 0; i < -number2; i++)
+                    {
+                        result += -number1;
+                    }
+                }
+                Console.WriteLine($"Multiply result: {result}");
+
+                Console.WriteLine("Do you want to proceed? Y/N");
+                string proceed = Console.ReadLine();
+                if (proceed.Equals("N") || proceed.Equals("n"))
+                    break;
+                else
+                    continue;
+            }
+        }
+
+        public static void SquareNumber() // 10
+        {
+            while (true)
+            {
+                Console.Write("Enter the number: ");
+                string stringNumber = Console.ReadLine();
+                bool isNumber = Int32.TryParse(stringNumber, out int number);
+                if (!isNumber || number <= 1)
+                {
+                    Console.WriteLine("Not a numeric, natural value or value is 1. Try again...");
+                    continue;
+                }
+
+                int result = 0;
+                for (int i = 1; i <= number; i++)
+                {
+                    result = i * i;
+                    if (result <= number)
+                        Console.WriteLine($"Square of {i} = {result}");
+                }
+
+                Console.WriteLine("Do you want to proceed? Y/N");
+                string proceed = Console.ReadLine();
+                if (proceed.Equals("N") || proceed.Equals("n"))
+                    break;
+                else
+                    continue;
+            }
+        }
+
+        public static void Fibonacci() // 11
+        {
+            while (true)
+            {
+                Console.Write("Enter the number: ");
+                string stringNumber = Console.ReadLine();
+                bool isNumber = Int32.TryParse(stringNumber, out int number);
+                if (!isNumber || number < 0)
+                {
+                    Console.WriteLine("Not a numeric values or less than 0. Try again...");
+                    continue;
+                }
+                int firstNumber = 0;
```

Comment:
The variable should be declared as much near as possible to the places where it is used.